source/develop/release-management/features/infra/link-following.html.md

```diff
@@ -0,0 +1,114 @@
+---
+title: API Link Following
+category: feature
+authors: oliel
+wiki_category: Feature
+wiki_title: API Link Following
+wiki_revision_count: 0
+wiki_last_updated: 2017-7-7
+feature_name: Link Following
+feature_modules: api (engine)
+feature_status: Released
+---
+
+## Feature Name
+
+API Link Following
+
+## Summary
+
+This features enable oVirt API users to request that the contents of some of the entity's links be returned inline, inside the requested entity.
+
+## Owner
+
+*   Name: Ori Liel (oliel)
+
+*   Email: <oliel@redhat.com>
+
+## Detailed Description
+
+Motivation:
+
+Currently when there is the need to retrieve multiple related objects from the API the only alternative is to retrieve the first one, and then, send additional requests to retrieve the related objects. For example, if you need a virtual machine and also the disks and NICs you need first to send a request like this:
```

Comment:
Corrections / suggestions:
"Currently, when there is a need to retrieve multiple related objects from the API, the only option is to retrieve the first one, and then send additional requests to retrieve the related objects. For example, if you need a virtual machine and also the disks and NICs, first you need to send a request like this:"